html/capture-selion-events.html

```diff
@@ -0,0 +1,37 @@
+<div class="selion-content-div">
+  <h1 id="what-is-events" class="page-header">Working with SeLion events</h1>
+
+  <p class="lead">
+    How to hook into events provided by SeLion.
+  </p>
+
+  <h2 id="introduction">Introduction</h2>
+
+  <p>
+    SeLion events are primarily those which are related to some action that you perform through elements provided by SeLion such as "click" (or) "select" etc.
+  </p>
+
+  <h2 id="getting-started">Getting Started with Event Handling in SeLion</h2>
+
+  <p>
+    SeLion offers support to hook into events that are generated by SeLion elements. These steps can be followed to start hooking into
+    events using SeLion. As an example we are going to create a DebugScreenshotListener which adds screenshots, that show data that was inputted to the page.
```

Comment:
Some minor edits need to happen on this text;
 
`As an example we are going to create a DebugScreenshotListener which adds screenshots, that show data that was inputted to the page.`

Could be

`As an example we are going to create a <code>DebugScreenshotListener</code> which takes a screenshot and adds it to the report before any element is clicked on.`